spec/types_spec.rb

```diff
@@ -89,6 +89,16 @@
     it { expect(type.describe).to eq('Set<String>') }
   end
 
+  describe 'Proc' do
+    subject(:type) {
+      Parlour::Types::Proc.new([], 'void')
+    }
+
+    it { expect(type.generate_rbi).to eq('T.proc.returns(void)]') }
```

Comment:
Looks like there's a typo in the expected output here - the stray `]` on the end